content/en/events/upcoming-events/gsoc-2025.md

```diff
@@ -67,339 +67,177 @@ To participate in GSoC with Kubeflow, you __must__ meet the GSoC [eligibility re
     - Read the [Introduction to Kubeflow](/docs/started/introduction/)
     - Review the [Architecture Overview](/docs/started/architecture/)
     - Consider [trying out Kubeflow](/docs/started/installing-kubeflow/) (not required, can be challenging)
-4. Review the [project ideas](#project-ideas-for-2025-gsoc) to decide which ones you are interested in:
+4. Review the [project ideas](#project-ideas) to decide which ones you are interested in:
     - You may wish to attend the next [community meeting](/docs/about/community/#kubeflow-community-calendar) for the group that is leading your chosen project.
     - ___NOTE:__ while we recommend you submit a proposal based on the project ideas, you can also submit a proposal with your own idea._
 5. Submit a proposal through the [GSoC website](https://summerofcode.withgoogle.com/) between __March 24th__ and __April 8th__.
 6. Wait for the results to be announced on __May 8th__.
 
-## Project Ideas for 2025 GSoC
+## Project Ideas
 
+### Project 1: Kubeflow Platform Enhancements
 
-### Project 1
-### Project Title: multi-tenancy/security, Kubeflow repository Migration, and CI/CD Enhancement
+__Components:__ Kubeflow Manifests, Kubeflow Dashboard, Kubeflow Notebooks, Kubeflow Pipelines
 
-#### Component: Kubeflow
+__Possible Mentors:__ [`@juliusvonkohout`](https://github.com/juliusvonkohout), [`@thesuperzapper`](https://github.com/thesuperzapper)
 
-#### Detailed Description:
+__Difficulty:__ Hard
 
-- Productionize the seaweedfs PoC as secure minio replacement https://github.com/kubeflow/manifests/tree/master/contrib/seaweedfs, since we are stuck with a 5 year old minio due to AGPL
-- add multi tenancy to seweadfs similar to the approach in feat(backend): isolate artifacts per namespace/profile/user using only one bucket pipelines#7725
-- Help to migrate Kubeflow/kubeflow components to kubeflow/dashboard and kubeflow/notebooks. This might even start before Gsoc
-- Renovate kubeflow/dashboard dependencies, addressing CVEs.
-- Implement architectural changes as needed.
-- Adapt and improve scripts and CI/CD in kubeflow/manifests, including matrix calls to test multiple Kubernetes versions simultaneously.
+__Size:__ 350 hours
 
-#### Expected Outcomes:
+__Possible Projects:__
 
-Successful migration of Kubeflow components.
-Hard multi-tenancy for the object storage
-Updated dependencies with resolved CVEs.
-Enhanced CI/CD pipeline supporting multiple Kubernetes versions.
+- Pipelines: productionize the [SeaweedFS PoC](https://github.com/kubeflow/manifests/tree/master/contrib/seaweedfs) as secure minio replacement
+- Pipelines: isolate artifacts per namespace/profile/user using only one bucket ([`kubeflow/pipelines#4649`](https://github.com/kubeflow/pipelines/issues/4649))
```

Comment:
Shouldn't this be a separate project since this is very complex tasks ?

Reply:
I think the idea is that @juliusvonkohout and myself will only mentor 1-2 of these because there are too many to accept a project for all.

They also have very similar skill requirements, so it's not a big deal, as it lets the students apply to the area they want to focus on.